images/ansible-operator/pipfile.Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM registry.access.redhat.com/ubi8/ubi:8.9-1107 AS basebuilder
+
+# Install Rust so that we can ensure backwards compatibility with installing/building the cryptography wheel across all platforms
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+ENV PATH="/root/.cargo/bin:${PATH}"
+RUN rustc --version
+
+# Copy python dependencies (including ansible) to be installed using Pipenv
+COPY ./Pipfile ./
+# Instruct pip(env) not to keep a cache of installed packages,
+# to install into the global site-packages and
+# to clear the pipenv cache as well
+ENV PIP_NO_CACHE_DIR=1 \
+    PIPENV_SYSTEM=1 \
+    PIPENV_CLEAR=1
+# Ensure fresh metadata rather than cached metadata, install system and pip python deps,
+# and remove those not needed at runtime.
+RUN set -e && yum clean all && rm -rf /var/cache/yum/* \
+  && yum update -y \
+  && yum install -y libffi-devel openssl-devel python39-devel gcc python39-pip python39-setuptools \
+  && pip3 install --upgrade pip~=23.3.2 \
+  && pip3 install pipenv==2023.11.15 \
```

Comment:
It it possible that these could be ENVs/ARGs, so that it's clearer that they need to be changed, and kept insync with the other dependency files?

Reply:
Since this is like this in other files in this repo, we can keep it this way in this PR and do a cleanup/refactor across the entire repo for things like this.

Reply:
Captured this as #74